gax-java/gax/src/main/java/com/google/api/gax/rpc/StubSettings.java

```diff
@@ -544,6 +544,11 @@ public ApiClock getClock() {
       return clock;
     }
 
+    /** Gets the custom Universe Domain that was previously set on this Builder */
+    public String getUniverseDomain() {
```

Comment:
This is for best practices but not actually used?

Reply:
This was to keep consistency with the other getters in the Builders. I don't think there is much use for this getter inside the Builder, but I didn't want to be inconstant with the fields for public facing classes (in case there is a user that has a use case for getting the values in the Builder).

I was on the fence about adding it in. If you think we shouldn't add it, I'm ok to remove it as well.

Reply:
Removed this.